maintainers/maintainer-list.nix

```diff
@@ -15294,6 +15294,11 @@
     github = "unrooted";
     githubId = 30440603;
   };
+  unsolvedcypher = {
```

Comment:
Please split this out into its own commit with the message `maintainers: add unsolvedcypher`

Reply:
Done!